photo-portfolio/src/main/java/com/throughapinhole/photoportfolio/Classes/Post.java

```diff
@@ -0,0 +1,81 @@
+package com.throughapinhole.photoportfolio.Classes;
+
+import java.util.ArrayList;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter @Setter @NoArgsConstructor
```

Comment:
Best to have one annotation per line just to make things clearer

Reply:
Done